testbed/eks/lib/testbed-stack.ts

```diff
@@ -0,0 +1,69 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as eks from '@aws-cdk/aws-eks';
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as cdk from '@aws-cdk/core'
+import { FluxV2, Repository } from './addons/fluxv2';
+
+export interface TestbedProps extends cdk.StackProps {
+  fluxRepos: Repository[];
+  fluxVersion: string;
+}
+
+export class TestbedStack extends cdk.Stack {
```

Comment:
Thoughts on just moving this whole thing up to `/testbed/stack.ts` to avoid layers of indirection and stuttering? I'd personally toss any notion of `lib`, since I don't expect us to be writing many tests. 

I'd also separate this stack and its properties like fluxVersion and repositories into a single file, and then build a top level construct that encapsulates the rest of the things. Not only is this a cleaner separation of concerns, it also allows allows customers to define their own stack that consumes this construct, since you can't wrap a stack within another stack.